README.md

```diff
@@ -4,45 +4,133 @@ This standalone project is designed to be deployed on a user's own server, enabl
+After this **Approver app** is approved by the owner and pairing process is completed, than, the handling of pending transactions flow may proceed naturally.
 
 ### First Run
 
-During the first run, a pairing process is initiated. This includes:
+There is difference between the first run of the **Approver app** and versus recurring runs.
 
-1. Logging into Inabit using the provided login token.
+Assuming you received all the details from our representatives, and You start the **Approver app** docker as described, so then
+During the first run, a pairing process is initiated. The **Approver app** will do these steps:
+
+1. Logging into Inabit using the provided login token, that you will receive from our team.
 2. Validating the user's current state.
 3. Requesting a pairing token.
 4. Producing an encrypted key used for signing, saving it to a mapped volume.
-5. Sending pairing data to Inabit and requesting the 'Owner' to authorize the approver.
+5. Sending pairing data to Inabit and requesting the 'Owner' to authorize the approver. **After this step, you will need to get the the _pairing token_ from the docker logs, and to send it to the owner of your Inabit account**.
 6. Refreshing the login token i.e. acquiring automatically a new valid login token.
 
+
+> If anything goes wrong during the initiation stage, the approver initiation process will be aborted.
+
+
 ### Recurring Runs
 
 For subsequent runs, after validating that the user is paired, the application continues its operation as usual.
 
-### Initiation Errors
-
-If anything goes wrong during the initiation stage, the approver initiation process will be aborted.
 
 ## Prerequisites
 
-1. An initial login token of a valid user within Inabit with the role: 'apiSigner'.
-2. Enabled communication with the Inabit server (valid URL and whitelisted IPs and ports).
+1. You must have an active account at `inabit.com`.
+
+2. In that account, there must be 2 existing users, with the following roles:
+  A. `apiAdmin` user.
+  B. `apiSigner` user.
+  
+This users should be  provided by our representatives.
+
+3. You have received a *login token* that is related to an "apiSigner" user. This token should have been provided to you by our representatives.
+ 
+4. Please verify that host can access `api.inabit.com`  server (whitelisted IPs and ports).
+
+5. Please be aware that for completion of the process, you will need interact with your inabit account owner.
+
+## Quick Setup
+
+Before you begin, ensure all prerequisites are met, and that you have the *login token* provided by our representatives.
+
+Then, run this in terminal (linux).
+
+```shell
+# clone this repository
+git clone https://github.com/In-a-bit/inabit-remote.git
+cd inabit-remote
+# create 2 needed directories
+mkdir refresh
+mkdir dat
+# save the login token in file.
+# in the next line, replace <login token> with the token you got.
+echo "<login token>" > refresh/r.dat
+touch dat/k.dat
```

Comment:
k.dat is suppose to be created on the fly.

Reply:
removed.